react/modules/forms/README.md

```diff
@@ -0,0 +1,93 @@
+# [React](https://github.com/rolling-scopes-school/tasks/tree/master/react) Forms 🌟
+
+## Module Overview 📚
+
+The "React Form" module explores the most popular form libraries for React are **React Hook Form** or **Formik**. Through this module, students will learn an overview of solutions to handle forms. The module covers one of the most popular solutions to create a fully functional form with input validation.
```

Comment:
We will be mostly using the same practical task for Forms which we've used before.
The task gives some hands-on experience working with uncontrolled components approach when dealing with forms, as well as React Hook Form.
May be it is worth providing more resources specific for those topics and leave just a few devoted to Formik (as an extra topic)?
Topics used in the practical task:
- Uncontrolled components when using forms;
- React Hook Form;
- Yup for validation;

Link to the old module:
https://github.com/rolling-scopes-school/tasks/tree/master/react/modules/_archive/module06

Reply:
Added links from old module.
Formik moved to additional materials.